pom.xml

```diff
@@ -40,6 +40,10 @@
   <profiles>
     <profile>
       <id>build-individual-bundles</id>
+      <properties>
```

Comment:
You should not compare Y-build with I-builds as this is what will happen if you merge this in master.